src/components/Dissolution/AssociationDetails.vue

```diff
@@ -27,8 +27,8 @@
         <v-col cols="12" sm="4" class="pr-4 pt-4 pt-sm-0">
           <label class="mailing-address-header">Mailing Address</label>
           <MailingAddress
-            v-if="!isEmptyAddress(getBusiness.officeAddress.mailingAddress)"
```

Comment:
`getBusiness` was only ever used to get tge office address, so I replaced the general getter with a more specific one.